assets/js/modules/analytics-4/components/dashboard/EnhancedMeasurementActivationBanner/SetupBanner.js

```diff
@@ -100,6 +106,12 @@ export default function SetupBanner( {
 		onSubmitSuccess();
 	}, [ setValues, submitChanges, onSubmitSuccess ] );
 
+	useMount( () => {
```

Comment:
I know this is how it was specced out in IB but there is a special component that is responsible for surveys triggering: `SurveyViewTrigger`. We need to use it instead of manually doing it like here to avoid having duplicate logic to trigger surveys across multiple components. So, instead of using this `useMount` hook we need to render the `SurveyViewTrigger` component:

```js
return (
    <BannerNotification ...>
        <SurveyViewTrigger triggerID="view_enhanced_measurement_cta" ttl={ DAY_IN_SECONDS } />
        ...
    </BannerNotification>
);
```